templates/definition/meter/sonnenbatterie_eco56.yaml

```diff
@@ -11,6 +11,9 @@ params:
     default: 7979
   - name: capacity
     advanced: true
+  - name: chargepower
+    default: 3000
```

Comment:
The maximum power is also available from the API under `S65`, so it would be better if there was a way to fetch that default and store it into a variable.
Is there a way to do this with templates?

Reply:
Siehe https://github.com/evcc-io/evcc/pull/20451 (https://github.com/evcc-io/evcc/pull/20451/files#diff-0b0f0388fa0ea5962b68de7565e1131be1912c96e6d198ad57f6b53d77775b17). Leider hat das Go Plugin ein Speicherleck.

Reply:
Soll ich diesen PR dementsprechend umbauen? Bzw. gibt es zu dem memory leak schon eine diskussion irgendwo?